### PR TITLE
Simple response parser added

### PR DESCRIPTION
**Pull request by chupAkabRRa**

### CODE REVIEW

File: ConfreaksMTV/FTResponseParser.m [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+#import "FTResponseParser.h"
+
+@implementation FTResponseParser
+
++ (NSArray<FTConference*> *)parseConferences:(NSArray *)conferences {
+    NSMutableArray<FTConference*> *array = [NSMutableArray array];
+    
+    if (!array) {
+        return nil;
```
Comment (eXiga) [inline ConfreaksMTV/FTResponseParser.m]: @chupAkabRRa what is the purpose of this check?

File: ConfreaksMTV/FTResponseParser.m [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    }
+    
+    return array;
+}
+
++ (NSArray<FTEvent*> *)parseEvents:(NSArray *)events {
+    NSMutableArray<FTEvent*> *array = [NSMutableArray array];
+    
+    if (!array) {
+        return nil;
```
Comment (eXiga) [inline ConfreaksMTV/FTResponseParser.m]: @chupAkabRRa what is the purpose of this check?

File: ConfreaksMTV/FTResponseParser.m [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    }
+    
+    return array;
+}
+
++ (NSArray<FTPresenter*> *)parsePresenters:(NSArray *)presenters {
+    NSMutableArray<FTPresenter*> *array = [NSMutableArray array];
+    
+    if (!array) {
+        return nil;
```
Comment (eXiga) [inline ConfreaksMTV/FTResponseParser.m]: @chupAkabRRa what is the purpose of this check?

File: ConfreaksMTV/FTResponseParser.m [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    }
+    
+    return array;
+}
+
++ (NSArray<FTVideo*> *)parseVideos:(NSArray *)videos {
+    NSMutableArray<FTVideo*> *array = [NSMutableArray array];
+    
+    if (!array) {
+        return nil;
```
Comment (eXiga) [inline ConfreaksMTV/FTResponseParser.m]: @chupAkabRRa what is the purpose of this check?

File: ConfreaksMTV/FTResponseParser.m (hunk 1 above)
Comment (chupAkabRRa) [inline ConfreaksMTV/FTResponseParser.m]: No memory exception. Not necessary in objc?

**Comment by eXiga** [comment]
@chupAkabRRa looking good. Merged.

**Pull request merged**